fe/fe-core/src/main/java/com/starrocks/alter/AlterJobMgr.java

```diff
@@ -878,7 +880,7 @@ private void processSwap(Database db, OlapTable origTable, List<AlterClause> alt
         String origTblName = origTable.getName();
         String newTblName = clause.getTblName();
         Table newTbl = db.getTable(newTblName);
-        if (newTbl == null || !newTbl.isOlapOrCloudNativeTable()) {
+        if (newTbl == null || !(newTbl.isOlapOrCloudNativeTable() || newTbl.isMaterializedView())) {
```

Comment:
How about `isCloudNativeMaterializedView`? we have too many table types...

Reply:
but that's different things:
1. `isMaterializedView`: `CLOUD_NATIVE_MATERIALIZED_VIEW`, `MATERIALIZED_VIEW`
2. `isOlapOrCloudNativeTable`: `OLAP`, `CLOUD_NATIVE`

and current code means, this item could either be a table, or an mv